package.json

```diff
@@ -17,6 +17,7 @@
     "babel-loader": "6.2.4",
     "babel-preset-es2015": "6.6.0",
     "babel-preset-react": "6.5.0",
+    "cpy": "^6.0.0",
     "css-loader": "0.23.1",
```

Comment:
Why is this file even here, in root? And package name is `dnn-tree-control`?

Reply:
maybe some wild cut and paste :)